iphone/Maps/LocalizedStrings/uk.lproj/Localizable.stringsdict

```diff
@@ -53,9 +44,11 @@
 			<string>NSStringPluralRuleType</string>
 			<key>NSStringFormatValueTypeKey</key>
 			<string>d</string>
-			<key>other</key>
+			<key>zero</key>
+			<string>%d треків</string>
+			<key>many</key>
```

Comment:
Is there a logical difference between "other" and "zero->many"?

Reply:
It depends on a language. Only some languages require "many" or "two" variants.